pkg/policy/policyUtils.go

```diff
@@ -0,0 +1,1281 @@
+package policy
+
+import (
+	"fmt"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+//there is no way for creating an array as constant, so creating a variable
+//this is the nearest to a constant on arrays.
+var TppKeyType = []string{"RSA", "ECDSA"}
+var TppRsaKeySize = []int{512, 1024, 2048, 3072, 4096}
+var CloudRsaKeySize = []int{1024, 2048, 4096}
+var TppEllipticCurves = []string{"P256", "P384", "P521"}
+
+func GetFileType(f string) string {
+	extension := filepath.Ext(f)
+
+	//As yaml extension could be yaml or yml then convert it to just .yaml
+	if extension == ".yml" {
```

Comment:
if the extension is in uppercase, then maybe will be great to support it

Reply:
fixed